test/test.js

```diff
@@ -14,7 +14,12 @@ require('../glpk.js').then(glpkjs => {
 
 			let json = JSON.parse(fs.readFileSync(`${__dirname}/data/${d}.json`).toString());
 			let prob = new glpk.Problem();
-			let z1, z2 = glpkjs.solve(json, glpkjs.GLP_MSG_ERR).result.z;
+			const settings = {
+        msgLev: glpkjs.GLP_MSG_ERR,
```

Comment:
the indentation here seems off.

Reply:
also, I made a comment around default values and what happens when some of the values are not set. We should decide if we make them optional or not.